pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -204,14 +258,17 @@ func (n *NodeLabeller) patchNode(originalNode, node *v1.Node) error {
 
 // prepareLabels converts cpu models + features to map[string]string format
 // e.g. "/cpu-model-Penryn": "true"
-func (n *NodeLabeller) prepareLabels(cpuModels []string, cpuFeatures map[string]bool) map[string]string {
+func (n *NodeLabeller) prepareLabels(cpuModels []string, kvmInfoLabels, cpuFeatures map[string]bool) map[string]string {
 	newLabels := make(map[string]string)
 	for key, value := range cpuFeatures {
 		newLabels["/cpu-feature-"+key] = strconv.FormatBool(value)
 	}
 	for _, value := range cpuModels {
 		newLabels["/cpu-model-"+value] = "true"
 	}
+	for key := range kvmInfoLabels {
```

Comment:
this is a global map, it's not technically thread safe to read from it like this.